iceoryx_examples/singleprocess/README.md

```diff
@@ -14,6 +14,8 @@ The example can be started with
 ./build/iceoryx_examples/singleprocess/single_process
 ```
 
+<!-- @todo Replace this with asciinema recording before v1.0-->
+
```

Comment:
Currently the output is always Sending/Receiving: **1**
But this should probably be fixed in the example and not here.

Reply:
I just tried  and the output is correct.